modules/toolbox/ml_runners.py

```diff
@@ -0,0 +1,19 @@
+# This file has functions which call different machine learning algorithms and handle results.
+import sys
+import os
+sys.path.insert(0, os.path.abspath('../..'))
```

Comment:
Yes, it's setup for each file individually. Not a pretty solution but I couldn't find a better one.